### PR TITLE
new T-SQL linter

### PR DESCRIPTION
**Pull request by jmbeach**
Made checker file for [tsqlint/tsqlint](https://github.com/tsqllint/tsqllint).

### CODE REVIEW

File: syntax_checkers/sql/tsqllint.vim [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+let g:loaded_syntastic_sql_tsqllint_checker = 1
+
+if exists('g:syntastic_sql_tsqllint_sort')
+    let g:syntastic_sql_tsqllint_sort = 1
+endif
+
+let s:save_cpo = &cpo
+set cpo&vim
+
+function! SyntaxCheckers_sql_tsqllint_IsAvailable() dict
```
Comment (lcd047) [inline syntax_checkers/sql/tsqllint.vim]: `IsAvailable()` is not needed.

File: syntax_checkers/sql/tsqllint.vim [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    return SyntasticMake({
+        \ 'makeprg': makeprg,
+        \ 'errorformat': errorformat })
+endfunction
+
+call g:SyntasticRegistry.CreateAndRegisterChecker({
+    \ 'filetype': 'sql',
+    \ 'name': 'tsqllint',
+    \ 'exec': 'tsqllint'})
```
Comment (lcd047) [inline syntax_checkers/sql/tsqllint.vim]: `exec` is not needed.

**Comment by jmbeach** [comment]
Removed unneeded items

**Comment by lcd047** [comment]
Thank you.  Merged, with this patch on top: 2ac540a1.

**Pull request merged**